code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -222,9 +222,8 @@
 /datum/chemical_reaction/mix_virus
 	name = "Mix Virus"
 	id = "mixvirus"
-	results = list(/datum/reagent/blood = 1)
-	required_reagents = list(/datum/reagent/consumable/virus_food = 1)
-	required_catalysts = list(/datum/reagent/blood = 1)
```

Comment:
Keep the catalyst, just remove the results. I suspect that doing this may delete the original blood and its data when mixed with amount of virus food equal to or greater than the blood reagent's.

Reply:
Thought it was the other way around, alright.